src/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const CB = {
+  INELIGIBLE: -1,
+  PENDING_SYNC: -10, // waiting for value from overlord
+  PENDING_COMPUTE: -15, // value from overlord set, waiting for local computation or refresh
+  PENDING_CLOSE: -20 // value from overlord set, waiting for proposal close for final computation
+};
+
+export const CURRENT_CB = parseInt(process.env.LAST_CB ?? '1');
```

Comment:
I was thinking actually the final cb could be always `1`. In case we want to resync we can simply update the entries we want resync with a PENDING_SYNC cb, we dont need to change the final cb necessary.